packages/aws-lambda/src/ssm.js

```diff
@@ -49,52 +47,46 @@ module.exports.init = ({ logger }) => {
 
   try {
     /**
-     * From https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html:
-     *
-     * Your code runs in an environment that includes the AWS SDK for JavaScript,
-     * with credentials from an AWS Identity and Access Management (IAM) role that you manage.
+     * As per AWS Lambda Node.js documentation: https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html
+     * The environment includes the AWS SDK for JavaScript, with credentials from an IAM role that you manage.
      */
-    // eslint-disable-next-line instana/no-unsafe-require, import/no-extraneous-dependencies
-    AWS = require('aws-sdk');
+    // eslint-disable-next-line import/no-extraneous-dependencies, instana/no-unsafe-require, prefer-const
+    const { SSMClient, GetParameterCommand } = require('@aws-sdk/client-ssm');
```

Comment:
In SDK v3, instead of requiring the entire `aws-sdk`,  imports only the necessary components: SSMClient and GetParameterCommand from `@aws-sdk/client-ssm`